### PR TITLE
Extracted models to a separate directory

### PR DESCRIPTION
**Pull request by AphonicChaos**
# Description
Reorganized models into their own folder to allow models and views to be worked on separately.

# Testing
1. Checkout this branch `git checkout project-reorganization`
2. Start the server: `npm start`
3. It should function as it did before. Namely:
- you can fill out a name, email, attending, and number of guests and submit
- you should still see the confirmation page with a link to guest list
- guest list should still add the given user to the list of attendees

### CODE REVIEW

File: models/response.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+
+// Model
+const responseSchema = mongoose.Schema({
```
Comment (swkane) [inline models/response.js:4]: Why did you change this to a const?

File: models/response.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    name: String,
+    email: String,
+    attending: Boolean,
+    guests: Number
+});
+
+// Instance
+const Response = mongoose.model('Response', responseSchema);
```
Comment (swkane) [inline models/response.js:12]: And this one?

**Review by swkane — CHANGES_REQUESTED** [review]

File: models/response.js (hunk 1 above)
Comment (AphonicChaos) [reply models/response.js:4]: Sorry, should have called this out explicitly. Schemas and models never change after being defined, so it made more sense to define as a `const` to ensure that they don't get changed accidentally.

**Review by swkane — APPROVED** [review]
Great job!
